packages/react-ui/storybook/stories/common/Palette.stories.js

```diff
@@ -142,25 +148,15 @@ const ActionTemplate = () => {
   );
 };
 
-const GreyTemplate = () => {
+const CartoBrandTemplate = () => {
   const classes = useStyles();
 
   return (
     <Grid container className={classes.container}>
-      <ColorBox colorVariant={'grey'} colorName={'900'} />
-      <ColorBox colorVariant={'grey'} colorName={'800'} />
-      <ColorBox colorVariant={'grey'} colorName={'700'} />
-      <ColorBox colorVariant={'grey'} colorName={'600'} />
-      <ColorBox colorVariant={'grey'} colorName={'500'} />
-      <ColorBox colorVariant={'grey'} colorName={'400'} />
-      <ColorBox colorVariant={'grey'} colorName={'300'} />
-      <ColorBox colorVariant={'grey'} colorName={'200'} />
-      <ColorBox colorVariant={'grey'} colorName={'100'} />
-      <ColorBox colorVariant={'grey'} colorName={'50'} />
-      <ColorBox colorVariant={'grey'} colorName={'A100'} />
-      <ColorBox colorVariant={'grey'} colorName={'A200'} />
-      <ColorBox colorVariant={'grey'} colorName={'A400'} />
-      <ColorBox colorVariant={'grey'} colorName={'A700'} />
+      <ColorBox colorVariant={'cartoBrand'} colorName={'naviBlue'} />
```

Comment:
If you're ok with naming, I'm ok, it's just that 'cartoBrand' vs cartoTheme feel a bit like strange in the naming

Reply:
Changed, thanks for the suggestion!